### ISSUE TITLE
kubedns Replica Set -> DaemonSet

### ISSUE THREAD

**Issue by jarkomes** [apprenda/kismatic#490]
I would advice to make de kubedns pod a Daemonset and let it scale on all nodes in the cluster, this to spread the load on deployments which rely heavy on internal DNS lookups.

**Comment by alexbrand** [comment]
What are your thoughts on ReplicaSet + Anti Affinity? This would allow different users to choose how wide they want to go when scaling KubeDNS. I am not sure if enforcing KubeDNS on all nodes is ideal.

**Comment by jarkomes** [comment]
That might actually be a good idea, I agree that it can be different from case to case. Not sure how to define it in regards to how kismatic works though.

**Comment by swade1987** [comment]
I agree with @alexbrand here as long as add Anti Affinity to the KubeDNS deployed we should be fine, having this as a DaemonSet seems a little overkill.
